app/models/patient.rb

```diff
@@ -10,6 +12,14 @@ class Patient
   field :secondary_person, type: String
 	field :secondary_phone, type: String
 
+  track_history   :on => [:name, :primary_phone, :secondary_person, :secondary_phone, :updated_by_id],
```

Comment:
There's an example on how to include it to track history on embedded documents on the gem readme. I can add that next and include it in the test I wrote.